Resources/Prototypes/_Moffstation/Entities/Objects/Devices/pinpointer.yml

```
@@ -0,0 +1,33 @@
+﻿- type: entity
+  parent: PinpointerBase
+  id: PinpointerPirate
+  name: pirate pinpointer
```

```suggestion
  name: pirate cove pinpointer
```
Just makes it a lil more clear